desktop-styles.css

```diff
@@ -504,3 +504,11 @@ button {
   border-radius: 8px;
   border: none;
 }
+
```

Comment:
- [ ] Your summary is not descriptive enough. You didn't added any details in your summary that what are you doing in this pull request. Kindly add some details what are you doing in this pull request :+1: